src/main/python/deep_qa/layers/backend/permute.py

```diff
@@ -0,0 +1,35 @@
+from typing import Tuple
+
+from keras import backend as K
+from keras.layers import Layer
+from overrides import overrides
+
+
+class Permute(Layer):
+    '''
+    This Layer calls K.permute_dimensions on both the input and the mask.
```

Comment:
double backticks on `Layer` (if applicable, see comments above) and `K.permute_dimensions`